lib/richTextResolvers.tsx

```diff
@@ -8,35 +8,47 @@ import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/solid";
 import { Block_ContentChunk } from "../models";
 import { ReactNode } from "react";
 
-export const createDefaultResolvers = (element: Elements.RichTextElement, renderRichText: (item: Block_ContentChunk) => ReactNode): Partial<PortableTextReactComponents> => ({
+export const createDefaultResolvers = (element: Elements.RichTextElement, renderRichText: (item: Block_ContentChunk) => ReactNode, insideTable: boolean = false): Partial<PortableTextReactComponents> => ({
```

Comment:
NITPICK: Maybe a little bit of tuning of insideTable variable... like `isImageInsideTable` or if more generic like `isElementInsideTable`

Reply:
`isElementInsideTable` it is

Reply:
`isElementInsideTable` it is